src/Microsoft.AspNet.Mvc.Xml/MvcOptionsExtensions.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNet.Mvc.Xml;
```

Comment:
actually this file is part of the separate xml package and not core...probably you missed the title of this file...